### DISCUSSION TITLE
keyboard shortcuts

### DISCUSSION THREAD

**Discussion by drhus** [standardnotes/forum#2567] [Suggestions]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes

**Is your feature request related to a problem? Please describe.**
Toward improving user experience, efficiency , using keyboard shortcuts instead of the mouse enables us to work faster, save us time and improving our productivity. I suggest adding 1) keyboard shortcuts 2) on mouse over keyboard shortcuts hint

**Describe the solution you'd like**
While the list can be extensive, particularly I miss Keyboard shortcuts for
Particularly for 
- [ ] Add Tag
- [ ] Create New Note 
- [ ] Focused writing
- [ ] Change Editor (while in plain text and you want to switch to markdown or others)
-
and Possibly additionally for:
- [ ] archive note
- [ ] protect note
- [ ] duplicate
- [ ] archive
- [ ] trash
- [ ] Pin/unpin
- [ ] prevent editing 
- [ ] open/close tag column
- [ ] open/close all note column 

**Describe alternatives you've considered**
while I believe this would be a great build in feature, we can develop as an extension

With Gratitude

**Comment by dasprive** [comment]
So... after doing some testing with Standard notes I just took the plunge, paid for a full year, and found out there are indeed no keyboard shortcuts. 

Looking at all the duplicate issues going years in the past, could someone from Standard notes add some context? How come there still aren't any keyboard shortcuts? I don't mean to sound demanding, but I agree with many of the others people that requested this feature: a note-taking app needs keyboard shortcuts.

**Comment by ErinPo** [comment]
+1
A note-taking app "for professionals" without keyboard shortcuts is really odd.

**Comment by marcolaux** [comment]
It would bring this app to a new level when I could just alt/cmd+tab to it, press ctrl/cmd+n, title gets focused so I can start typing the title, another tab get's me to the note content and I can start writing my thoughts down.

I currently have to use my mouse, move that all over 2 big screens, navigate and click on the + icon, navigate and click on the title, navigate and click on the content area. This is sadly not intuitive and costs time and thought-fragments.

This would make the app also more barrier-free (althouth this is a whole new chapter of course). But people with ADHD would help this a lot.

**Comment by ncolestock** [comment]
I'd encourage the developers to do this quickly. Anyone familiar with note-taking expects a keyboard shortcut to at least add a new note.

**Comment by atmoio** [comment]
Keyboard shortcuts for primary actions are now available in the latest version. Not yet possible to customize shortcuts but this is on our radar.

<img width="326" alt="Screen Shot 2022-11-21 at 9 52 22 AM" src="https://user-images.githubusercontent.com/3277844/203099678-8440795f-03c5-4af5-b75b-4df70f9eccda.png">
<img width="329" alt="Screen Shot 2022-11-21 at 9 52 03 AM" src="https://user-images.githubusercontent.com/3277844/203099683-dd61152e-18ec-4924-a834-7b936e713e82.png">
<img width="368" alt="Screen Shot 2022-11-21 at 9 51 55 AM" src="https://user-images.githubusercontent.com/3277844/203099685-b35a4810-8757-4733-b28f-b0a63bd16e6f.png">

**Comment by marcolaux** [comment]
@moughxyz thank you so much 🙏

But when the editor is focued (what might be the case >90% of the time) the shortcuts don't seem to work (Windows / web version).

**Comment by marcolaux** [comment]
Oh but they work in the new Super editor for me.

Would it be possible to also support the previous editors or will they become legacy at some point?

I can also close the menus when I click inside the Super editor 🥳🎉

**Comment by steel-steven-87** [comment]
> So... after doing some testing with Standard notes I just took the plunge, paid for a full year, and found out there are indeed no keyboard shortcuts.
> 
> Looking at all the duplicate issues going years in the past, could someone from Standard notes add some context? How come there still aren't any keyboard shortcuts? I don't mean to sound demanding, but I agree with many of the others people that requested this feature: a note-taking app needs keyboard shortcuts.

I loved the idea of Standard Notes but this is the main reason I never got the premium version and I've been looking for a replacement. It felt stupid to pay for an note taking application that doesn't manage to implement all of the basic features that windows notepad has... which is probably the most minimal editor I can think of.

**Comment by marcolaux** [comment]
> I loved the idea of Standard Notes but this is the main reason I never got the premium version and I've been looking for a replacement. It felt stupid to pay for an note taking application that doesn't manage to implement all of the basic features that windows notepad has... which is probably the most minimal editor I can think of.

It's in constant development and when a more basic feature is lacking, it means that the priorities were somewhere else. Like encryption, infrastructure. There was so much going on lateley that I can't keep up with the changes myself.
Ranting doesn't help anyone and comparing it with Notepad plays down the afford the team put into this. Every decision has a reason.

**Comment by ashutoshsaboo** [comment]
FYI, Keyboard shortcuts for primary actions are already live on Standard
Notes desktop/web. SN has gone through quite an upgrade in last few months,
read below a post on the recent features that they've shipped:

https://blog.standardnotes.com/40627/introducing-super-notes-moments-and-offline-file-access

**Comment by christian-bentley** [comment]
Surely CMD & N is a pre-requisite and a quick win to implement?

**Comment by effieeee** [comment]
Hi @christian-bentley, on Mac I believe the new note shortcut is Cmd + Shift + N, but this may or may not work on the web app depending on whether or not your browser has certain add ons that use the shortcut too 🙏 For Windows and Linux, it's Alt + Shift + N

**Comment by joshuaiz** [comment]
Cmd + Shift + N or Cmd + N does not work on Mac desktop app — or any other shortcuts for that matter.

Agreed this is a major deficiency in this app to not have basic keyboard shortcuts.

**Comment by dedfft** [comment]
Are there shortcuts for the Super editor? I feel like different shortcuts for highlighting, making text bold, adding a divider, adding a checkbox, and all that stuff, or converting something to something else, would be really, really helpful.
